art/metrics/privacy/membership_leakage_metrics.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from typing import TYPE_CHECKING, Optional
+
+import logging
+
+import numpy as np
+import scipy
+
+from art.utils import check_and_transform_label_format, is_probability
+from art.estimators.classification.classifier import ClassifierMixin
+
+if TYPE_CHECKING:
+    from art.estimators.classification import Classifier
+
+
+def PDTP(target_estimator: "Classifier", extra_estimator: "Classifier", x: np.ndarray, y: np.ndarray,
+         indexes: Optional[np.ndarray] = None, num_iter: Optional[int] = 10) -> np.ndarray:
+    """
+        Compute the pointwise differential training privacy metric for the given classifier and training set.
+        Taken from: https://arxiv.org/abs/1712.09136
+
+        :param target_estimator: The classifier to be analyzed.
+        :param extra_estimator: Another classifier of the same type as the target classifier, but not yet fit.
+        :param x: The training data of the classifier.
+        :param y: Target values (class labels) of `x`, one-hot-encoded of shape (nb_samples, nb_classes) or indices of
+                  shape (nb_samples,).
+        :param indexes: the subset of indexes of `x` to compute the PDTP metric on. If not supplied, PDTP will be
+                        computed for all samples in `x`.
+        :param num_iter: the number of iterations of PDTP computation to run for each sample. If not supplied,
+                         defaults to 10. The result is the average across iterations.
+        :return: an array containing the average PDTP value for each sample in the training set.
```

Comment:
Could you please add an additional sentence describing PDTP like, if I'm right, a higher value means more leakage?